Samples/Client/DirectxWin32/src/default_main_window.cpp

```diff
@@ -655,11 +655,11 @@ void DefaultMainWindow::CreateChildWindow(HWND* wnd, DefaultMainWindow::ChildWin
 void DefaultMainWindow::CreateChildWindows()
 {
 	// Create the child windows in tab order.
-	CreateChildWindow(&auth_uri_label_, AUTH_ID, L"Static", ES_LEFT | ES_READONLY, 0);
-	CreateChildWindow(&auth_uri_, AUTH_ID, L"Static", ES_CENTER | ES_READONLY, 0);
+	CreateChildWindow(&auth_uri_label_, AUTH_ID, L"Static", ES_CENTER | ES_READONLY, 0);
+	CreateChildWindow(&auth_uri_, AUTH_ID, L"Edit", ES_LEFT , 0);
```

Comment:
we probably need ES_READONLY on the edit control, right? https://msdn.microsoft.com/en-us/library/windows/desktop/bb775464(v=vs.85).aspx